currency_rate_update_boi/readme/CONFIGURE.rst

```diff
@@ -0,0 +1,9 @@
+**Italiano**
+
+Suggeriamo di eseguire l'azione pianificata "Currency Rates Update" alla fine del giorno,
```

Comment:
Dato che il nome dell'azione viene tradotto, meglio mettere direttamente il termine in italiano (che ho appena tradotto).
"Aggiornamento tassi di cambio (OCA) giornaliero" oppure "Aggiornamento tassi di cambio" (in base all'altro commento).

Reply:
Modificato